packages/ui/react-ui-navtree/src/components/NavTreeItem.tsx

```diff
@@ -195,6 +195,7 @@ export const NavTreeItem: MosaicTileComponent<NavTreeItemData, HTMLLIElement> =
                   testId={`navtree.treeItem.actionsLevel${level}`}
                 />
               )}
+              {renderPresence?.(node)}
```

Comment:
This appears to be a render function, for which we already have pretty solid patterns that you established. Maybe this prop should be called `PresenceComponent`, have a default maybe, and render here in JSX.

Reply:
IMO it should definitely be capitalized and rendered using JSX syntax so it’s clear what it does and aligns with the other parts of our API that do the same thing.